packages/astro-font/utils.ts

```diff
@@ -5,23 +5,26 @@ import { Buffer } from 'node:buffer'
 import { getFallbackMetricsFromFontFile } from './font.ts'
 import { pickFontFileForFallbackGeneration } from './fallback.ts'
 
-interface Record {
-  [property: string]: string
-}
+type GlobalValues = "inherit" | "initial" | "revert" | "revert-layer" | "unset"
+
+type FontWeight = 100 | 200 | 300 | 400 | 500 | 600 | 700 | 800 | 900
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#number

This indicates that the font weight ranges from 1 to 1000. Woah.